src/API/WP/OAuthService.php

```diff
@@ -159,16 +159,57 @@ public function revoke_wpcom_api_auth(): string {
 		$request = $this->container->get( Jetpack::class )->remote_request( $args );
 
 		if ( is_wp_error( $request ) ) {
+
+			/**
+			 * @event revoke_wpcom_api_auth
+			 */
+			do_action(
+				'woocommerce_gla_track_event',
+				'revoke_wpcom_api_auth',
+				[
+					'status'  => 'error',
+					'error'   => $request->get_error_message(),
+					'blog_id' => Jetpack_Options::get_option( 'id' ),
+				]
+			);
+
 			throw new Exception( $request->get_error_message(), 400 );
 		} else {
 			$body   = wp_remote_retrieve_body( $request );
 			$status = wp_remote_retrieve_response_code( $request );
 
 			if ( ! $status || $status !== 200 ) {
-				$data = json_decode( $body, true );
-				throw new Exception( $data['message'] ?? 'Error revoking access to WPCOM.', $status );
+				$data    = json_decode( $body, true );
+				$message = $data['message'] ?? 'Error revoking access to WPCOM.';
+
+				/**
+				 * @event revoke_wpcom_api_auth
+				 */
+				do_action(
```

Comment:
Same as https://github.com/woocommerce/google-listings-and-ads/pull/2452/files#r1666846943